synonymdev/bitkit-react-native
# fix(wallet): Refactor setupFeeForOnChainTransaction

### Description
- Moves `setupFeeForOnChainTransaction` usage to an earlier part of the tx creation flow.
- Updates `setupFeeForOnChainTransaction` to attempt 1 `satsPerByte` if `selectedFeeId` fails.

### Linked Issues/Tasks
- Attempts to address #1574

### Type of change

### Tests

### Screenshot / Video

https://github.com/synonymdev/bitkit/assets/8532651/10cc7ae0-23de-4cdb-bb4d-5b8b338b586a


https://github.com/synonymdev/bitkit/assets/8532651/f1582410-e3b8-4062-abd2-cdca9c3aea0e


### QA Notes
- Set a low spending amount such as 547 with multiple inputs/utxos and attempt to continue the tx creation flow. You should now get prompted with an alert suggesting that you increase the send amount. Or in the event you're on the coin selection screen, remove some of your UTXO's.
